src/main/java/seedu/mtracker/commands/FindCommand.java

```diff
@@ -8,25 +8,25 @@
 public class FindCommand extends Command {
     public static final String COMMAND_WORD = "find";
     public static final String EMPTY_STR = "";
-    private String keyword;
+    private String searchString;
     private ArrayList<Instrument> filteredInstruments;
 
     public FindCommand() {
-        keyword = EMPTY_STR;
+        searchString = EMPTY_STR;
     }
 
-    public void setKeyword(String searchString) {
-        keyword = searchString;
+    public void setSearchString(String searchString) {
+        this.searchString = searchString;
     }
 
-    public String getKeyword() {
-        return keyword;
+    public String getSearchString() {
+        return searchString;
     }
```

Comment:
Can remove this method if its not being used. If I recall it was not used at all

Reply:
Removed.